spec/visual/samples/magik

```diff
@@ -8,7 +8,8 @@ def_slotted_exemplar(
         :test_object,
         {
                 {:slot, _unset}
-        })
+        },
+        {_class |java.lang.Integer|)
```

Comment:
It's not causing a lexing problem but is there supposed to be a closing brace in the class declaration?

Reply:
Nice catch! Fixed.